apps/glusterfs/app_node.go

```diff
@@ -320,6 +321,49 @@ func (a *App) NodeDelete(w http.ResponseWriter, r *http.Request) {
 				return err
 			}
 
+			volumes, err := ListCompleteVolumes(tx)
```

Comment:
This feels like it should be it's own function, it's fairly long and conceptually independent from the surrounding code. Something like `refreshVolumeNodes(tx, clusterID)`